### DISCUSSION TITLE
Additional Meta fields

### DISCUSSION THREAD

**Discussion by matt-shirey** [Z3d0X/filament-fabricator#185] [Q&A] [answered]
I've added a new sidebar section for SEO fields and have it saving just fine.  The issue I'm running into is passing that info to the the head of the base layout.  I see the method of `pushMeta` but I don't know where to put that code to push data from my `$page` variable.

**Comment by Voltra** [comment, marked as answer]
That `pushMeta` API is mostly for static data like the sitename and what have you.

You can always override the base layout by publishing the views (or making the override manually if you only need that one file).

That's what I did when I was using [`ralphjsmit/laravel-filament-seo`](https://packagist.org/packages/ralphjsmit/laravel-filament-seo)
